src/Analyzers/CSharp/Analyzers/CSharpAnalyzersResources.resx

```diff
@@ -365,4 +365,10 @@
   <data name="Convert_to_top_level_statements" xml:space="preserve">
     <value>Convert to top-level statements</value>
   </data>
+  <data name="Convert_to_UTF8_string_literal" xml:space="preserve">
+    <value>Convert to UTF8 string literal</value>
```

Comment:
```suggestion
    <value>Convert to UTF-8 string literal</value>
```

Reply:
Looks like the compiler uses `UTF8` without the hyphen too:
https://github.com/dotnet/roslyn/blob/main/src/Compilers/CSharp/Portable/CSharpResources.resx#L7082-L7090

@AlekseyTs should these be "UTF8" or "UTF-8"? I'm not fussed, but we should be consistent. Happy to update those strings as part of this PR if required.

Reply:
Going to change them all to UTF-8, as thats what Kathleen used in the blog post :)